test/libp2p.spec.js

```diff
@@ -0,0 +1,432 @@
+'use strict'
+/* eslint-env mocha */
+
+const expect = require('chai').expect
+const libp2p = require('../src')
+// const PeerId = require('peer-id')
+const PeerInfo = require('peer-info')
+const multiaddr = require('multiaddr')
+const parallel = require('run-parallel')
+const bl = require('bl')
+const spawn = require('child_process').spawn
+const path = require('path')
+
+describe('libp2p-ipfs', () => {
+  let nodeA // TCP
+  let nodeB // TCP
+  let nodeC // TCP
+  let nodeD // TCP
+  let nodeE // TCP + WebSockets
+  let nodeF // WebSockets
```

Comment:
then I have to remember array indexes, this way I can look at it like a table :)